eng/Build.props

```diff
@@ -12,8 +12,14 @@
     <!-- Exclude VSIX projects on non-Windows -->
     <ProjectToBuild Condition="'$(OS)'!='WINDOWS_NT' and '$(SdkTaskProjects)'==''" Include="$(MSBuildThisFileDirectory)..\src\Razor\Razor.Slim.sln" />
 
-    <NodeJsProjects Include="$(RepoRoot)src\Razor\src\**\*.npmproj" RestoreInParallel="false"/>
-    <NodeJsProjects Include="$(RepoRoot)src\Razor\test\**\*.npmproj" RestoreInParallel="false" />
+    <NodeJsProjects Include="$(RepoRoot)src\Razor\eslint-config-razor-tooling\eslint-config-razor-tooling.npmproj" RestoreInParallel="false" />
```

Comment:
I ended up expanding these because I ran into an issue (seemingly with Arcade) where it was trying MSBuild the wildcarded directory instead of the projects found. I looked into why that was, but it wasn't progressing so I erred on the side of getting these improvements in. These projects don't change very often, so I don't see it as a big lose.